main.py

```diff
@@ -112,22 +114,53 @@ def markers(methods=["GET", "POST"]):
         marker.put()
         return make_response(json.dumps(marker.serialize(self.user)))
 
+#TODO: if not in future use - remove
 @app.route("/markers/(.*)", methods=["GET", "PUT", "DELETE"])
 @user_required
 def marker(self, key_name):
+    return(make_response(get_put_delete(self, key_name, Marker)))
+
+@app.route("/highlightpoints/", methods=['POST'])
```

Comment:
we are separating the user from location, as we don't want to save user and it's location, only the location is important.